pkg/manager/service.go

```diff
@@ -157,6 +157,11 @@ func (m *Manager) Delete(ctx context.Context, req *rpc.DeleteRequest) (resp *emp
 		if rmDirErr := os.RemoveAll(types.GetBackingImageDirectory(m.diskPath, req.Name, req.Uuid)); rmDirErr != nil {
 			log.WithError(rmDirErr).Errorf("Backing Image Manager: failed to remove the backing image work directory at the end of the deletion")
 		}
+		// Delete cmd is used to remove the tmp file left on the host as well when preparing backing image file failed.
+		tmpFilePath := fmt.Sprintf("%s%s", types.GetDataSourceFilePath(m.diskPath, req.Name, req.Uuid), types.TmpFileSuffix)
+		if rmTmpFileErr := os.RemoveAll(tmpFilePath); rmTmpFileErr != nil {
+			log.WithError(rmTmpFileErr).Errorf("Backing Image Manager: failed to remove the data source tmp file at the end of the deletion")
```

Comment:
Suggest using `Warnf` instead, because this error not returned but tolerated on purpose. 

The same as above the remove call.

Reply:
sure